tss-esapi/src/interface_types/algorithm.rs

```diff
@@ -292,8 +292,8 @@ impl TryFrom<TPMI_ALG_SYM_MODE> for SymmetricMode {
 /// Enum representing the asymmetric algorithm interface type.
 ///
 /// # Details
-/// Use [AsymmetricAlgorithmSelection] instead where possible.
-/// This corresponds to TPMI_ALG_ASYM.
+/// Use [crate::abstraction::AsymmetricAlgorithmSelection] instead where possible.
+/// This corresponds to `TPMI_ALG_ASYMq.
```

Comment:
```suggestion
/// This corresponds to `TPMI_ALG_ASYM`.
```

Reply:
Ugh, that was pretty embarrassing :sweat_smile: I wonder why didn't `cargo doc` complain about broken markup though... :thinking: 

One way or another -> I've applied that fix to the commit and rebased everything accordingly. Thanks for your eagle eyes! :eagle: